spec/services/spreadsheets/manufacturers_spec.rb

```diff
@@ -79,7 +79,7 @@ def expect_target_manufacturer(mnfg)
       end
       context "with no matching manufacturer" do
         let!(:manufacturer) { nil }
-        it "creates" do
+        it "creates", :flaky do
```

Comment:
Super annoying. Tried to figure it out twice - #2798 and #2799 but it still happens, so just mark it.

Reply:
Updated to create a manufacturer in the let block (rather than nil). Hopefully that solves the flakiness?

Reply:
didn't solve the flakiness. Marked it flaky